package.json

```diff
@@ -22,18 +22,18 @@
     "lint": "ng lint",
     "test": "jest --watchAll",
     "ci": "jest",
-    "build": "ng build --prod",
+    "build": "ng build --prod --build-optimizer",
     "serve": "node lib/server.js dist"
   },
   "private": true,
   "main": "index.ts",
   "dependencies": {
     "@angular/animations": "^4.0.0",
-    "@angular/cli": "^1.0.0",
+    "@angular/cli": "^1.4.2",
     "@angular/common": "^4.0.0",
     "@angular/compiler": "^4.0.0",
-    "@angular/compiler-cli": "^4.0.0",
-    "@angular/core": "^4.0.0",
+    "@angular/compiler-cli": "^4.4.3",
+    "@angular/core": "^4.4.3",
```

Comment:
Hmm... I _think_ you may need to upgrade all the other angular deps too?  Otherwise the component tests fail with weird errors (some mismatch in angular internals), and I also wonder about the impact on other angular stuff (router/animations/etc).  Which we don't use.  But maybe someday we will!

Reply:
Thanks! Should be fixed now.